_data/implementations.yml

```diff
@@ -76,14 +76,6 @@
   version: 1.6.1
   notes: "A front-end component library aimed at implementing USWDS design patterns in React."
 
-- name: us_web_design_standards gem
```

Comment:
Removed since this is confusing. We do maintain `uswds-jekyll` which has the same functionality, as far as I can determine